azurelinuxagent/common/protocol/wire.py

```diff
@@ -1424,24 +1421,12 @@ def parse(self, xml_text):
         self.role_name = getattrib(role, "name")
         deployment = find(xml_doc, "Deployment")
         self.deployment_name = getattrib(deployment, "name")
-        return self
 
 
 class SharedConfig(object):
-    """
-    parse role endpoint server and goal state config.
```

Comment:
Except that the comment is not actually telling you what these objects do... they neither parse the SharedConfig nor write to SharedConfig.xml. 

I decided to keep the verbose "Load foo.xml" messages in all these classes, since they do not load the xml files either, but I was very tempted to remove those as well.